### PR TITLE
*: admin check table return tidb specific error code.

### PR DESCRIPTION
**Pull request by winkyao**
### What problem does this PR solve? 

`ADMIN CHECK TABLE` return 1105 (Unknown) error before, it is difficult to distinguish the specific tidb error. This PR return 8003 which is TiDB specific error code to the client.  Then we can avoid testing the admin check failed with `strings.Contains(err.Error(), "1105")`.

### Check List 

Tests 

 - Unit test

Code changes

 - Has exported function/method change

### CODE REVIEW

**Comment by shenli** [comment]
LGTM

File: mysql/errcode.go [hunk 1]
```diff
@@ -894,6 +894,7 @@ const (
 	// TiDB self-defined errors.
 	ErrMemExceedThreshold = 8001
 	ErrForUpdateCantRetry = 8002
+	ErrAdminCheckTable    = 8003
```
Comment (jackysp) [inline mysql/errcode.go]: Do not forget to update the doc-cn.

**Comment by winkyao** [comment]
@jackysp PTAL

**Pull request merged**